### PR TITLE
added section to troubleshoot big project folders

### PR DESCRIPTION
**Pull request by thigg**
wip because it doesnt work yet. Also maybe this should be the default?

See https://forum.sailfishos.org/t/live-debugging-very-slow/12518/3

### CODE REVIEW

**Comment by thigg** [comment]
This works for me now!

File: Develop/Apps/Tutorials/QML_Live_Coding_With_Qt_QmlLive/README.md [hunk 1]
```diff
@@ -75,6 +75,17 @@ Let's say you would like to concentrate now on a single component from your appl
 
 If you develop for multiple devices, it is possible to run the application simultaneously on all the devices. In that case you will find useful the **All Hosts** widget. Dropping a document onto this widget influences all connected hosts. Another feature useful when working with multiple hosts is the **Follow** behavior. Activating a document in the workspace view (e.g. by double clicking it – depending on your platform) triggers active document change on all **Follow**-enabled hosts.
 
+## Troubleshooting
+
+#### Syncing changes is very slow / Stange output during syncing
```
Comment (vigejolla) [inline Develop/Apps/Tutorials/QML_Live_Coding_With_Qt_QmlLive/README.md:80]:
```suggestion
#### Syncing changes is very slow / Strange output during syncing
```

**Comment by thigg** [comment]
@vigejolla thanks and done

**Review by vigejolla — APPROVED** [review]
LGTM

**Comment by vigejolla** [comment]
Thank you!

**Pull request merged**